Makefile

```diff
@@ -131,6 +131,13 @@ test-sim-backwards-compatible: ## Test uploading a file with lastest release (je
 	@echo "Running ${@}"
 	@./scripts/test-sim-backwards.sh
 
+.PHONY: check-monitoring
+check-monitoring: ## Check for locked monkit calls that have changed
+	@echo "Running ${@}"
+	@go run ./scripts/check-monitoring.go | diff -U0 ./monkit.lock - \
```

Comment:
Will `diff` work for Windows users?

Reply:
Probably not, I can look into a cross-platform solution but do we expect all our make targets to work on windows? Do the shell scripts work on windows?